plugins/modules/rulebook_activation_copy.py

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
+
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+---
+module: rulebook_activation_copy
+short_description: Copy rulebook activations in the EDA Controller
+description:
+  - This module allows the user to copy rulebook activations in the EDA Controller.
+options:
+  name:
+    description:
+      - The name of the rulebook activation.
+    type: str
+    required: true
+  copy_from:
+    description:
+      - Name of the existing rulebook activation to copy.
+      - If set, copies the specified rulebook activation.
+      - The new rulebook activation will be created with the name given in the C(name) parameter.
+    type: str
+    version_added: 2.6.0
+  description:
+    description:
+      - The description of the rulebook activation.
+    type: str
+  organization_name:
+    description:
+      - The name of the organization. Required when state is present.
+      - This parameter is supported in AAP 2.5 and onwards.
+        If specified for AAP 2.4, value will be ignored.
+    type: str
+    aliases:
+      - organization
+  log_level:
+    description:
+      - Allow setting the desired log level.
+      - This parameter is supported in AAP 2.5 and onwards.
+        If specified for AAP 2.4, value will be ignored.
+    type: str
+    default: "error"
+    choices: ["debug", "info", "error"]
+  state:
+    description:
+      - Desired state of the resource.
+    default: "present"
+    choices: ["present", "absent"]
+    type: str
+extends_documentation_fragment:
+  - ansible.eda.eda_controller.auths
+notes:
+
+"""
+
+EXAMPLES = r"""
+- name: Copy an existing rulebook activation
+  ansible.eda.rulebook_activation_copy:
+    name: "Example Rulebook Activation - copy"
+    copy_from: "Example Rulebook Activation"
+    organization_name: "Default"
+
+"""
+
+RETURN = r"""
+id:
```

Comment:
The api returns the whole activation object, not just the id. Have you verified the return format in the collection?

Reply:
This is a thing that we should revisit at some point. Right not the collection only returns the id in all the module, delegating to its version "_info" module to fetch all the data. But I don't know if this is the standard, I see in other ansible modules that it returns the whole object. We might want to change this behavior for the whole collection but so far, returning the id would be inline with rest of the modules (if we are actually returning the id and not the whole object)